.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/smartdevicelink/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/LuxoftSDL/rpc_spec.git
```

Comment:
Note this should be reverted before merge

Reply:
@JackLivio I have some misunderstanding about the process that we are following with sub-modules. 
This is not special case -  many PR's make changes in rpc spec, so I would like to have some rules that we gona follow. 

For now I have following understanding : 
 0. *develop* branch should contain API version of the next release 
 1. If feature/fix  affects both **sdl_core** and **rpc_spec** repos, 2 PRs should be delivered.
 2. sdl_core PR should contain a **separate commit** with sub-module reference to rpc_spec PR
 3. After final approve, resolving merge conflicts and regression check sdl_core **separate commit** should be reverted and both PRs should be merged to develop simultaneously.

Could you please confirm that step **3** should be done by project maintainers?

Reply:
@LuxoftAKutsan Yes that is a good plan. I was only leaving the note to revert as a reminder after review is completed.